docs/src/operate/c8y/017_apama_software_management_plugin.md

```diff
@@ -59,7 +59,7 @@ For Apama projects:
 1. The version must be suffixed with `::apama` as in `1.0::apama` or just `::apama` if no  version number is necessary.
 2. The uploaded binary must be a `zip` file that contains the `project` directory. If a directory named `project` is not found by the plugin at the root level in the zip, it is considered invalid.
 
-![Add new apama project in Software Repository](./images/apama-plugin/apama-project-c8y-software-repository.png)
+![Add new apama project in Software Repository](../../howto-guides/images/apama-plugin/apama-project-c8y-software-repository.png)
```

Comment:
The `images` dir could be moved to the top level in a follow-up PR, so that we can get rid of the `how-to-guides` dir completely.